packer/ansible/roles/localsource/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: install dev libraries for NPM installs
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - libkrb5-dev
+    - unzip
+  sudo: yes
+
+- name: Install RackHD from local source
+  copy: src="../../{{ item }}" dest="{{ ansible_env.HOME }}/src/"
+  with_items:
+    - on-core
+    - on-tasks
+    - on-http
+    - on-tftp
+    - on-dhcp-proxy
+    - on-taskgraph
+    - on-syslog
+    - on-wss
+    - on-tools
+    - on-imagebuilder
+
+- name: Npm install Repos
+  npm: path="{{ ansible_env.HOME }}/src/{{ item }}"
+       production=yes
```

Comment:
There may be a reason for this.  Can you point me at the specific playbooks you are referring to?  I'll take a look.